lib/video_info/version.rb

```diff
@@ -1,3 +1,3 @@
 class VideoInfo
-  VERSION = '2.7.1'.freeze
+  VERSION = '2.7.2'.freeze
```

Comment:
Please do not update the version in PR, I will do it myself when a fix is released.